### PR TITLE
Create calc.py

### LINKED ISSUE #31: Implement calculator in python

### PR DESCRIPTION
**Pull request by shubhamnag14**
Simple calculate using python

### CODE REVIEW

**Comment by shubhamnag14** [comment]
kindly go through the program and let me know if any change is required

File: Python/calc.py [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+            print(num1, "+", num2, "=", add(num1, num2))
+
+        elif choice == '2':
+            print(num1, "-", num2, "=", subtract(num1, num2))
+
+        elif choice == '3':
+            print(num1, "*", num2, "=", multiply(num1, num2))
+
+        elif choice == '4':
+            print(num1, "/", num2, "=", divide(num1, num2))
```
Comment (ary1733) [inline Python/calc.py]: check whether num2 is zero or not.

**Comment by ary1733** [comment]
@shubhamnag14 if possible have a loop that goes until the user wants to exit

**Review by shubhamnag14 — COMMENTED** [review]
i have  updated the calc.py ..
please go thtough and check it

**Review by ary1733 — APPROVED** [review]

**Comment by shubhamnag14** [comment]
what the hack is going on..you have gave me duplicate label...if you think its duplicate then why did you merge this PR..the other has created the same thing..you have labeled him as hacktoberfest accepted..if you dont like my PR..the why did you accept my pr...accepted as well as labeled as duplicate..like really...this is my first pr..and you labelled as duplicate..i thought i will start my hacktoberfest challenge with one pr accepted and labeled hacktoberfest accepted..

**Comment by ary1733** [comment]
I have added hacktober accepted but please do submit issues before they get closed.

**Comment by ary1733** [comment]
@shubhamnag14  Enjoy hacktober fest and contribute as much as you can

**Pull request merged**